Remotion/Web/Development.WebTesting/WebTestFeatureCollection.cs

```diff
@@ -0,0 +1,82 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+
+namespace Remotion.Web.Development.WebTesting;
+
+/// <inheritdoc cref="IWebTestFeatureCollection" />
+/// <threadsafety static="true" instance="true" />
+/// <remarks>
+/// Features added to the <see cref="WebTestFeatureCollection"/> will be disposed when the collection is diposed.
```

Comment:
```suggestion
/// Features added to the <see cref="WebTestFeatureCollection"/> will be disposed when the collection is diposed.
```
sentence is weird / don't understand. please rewrite.